hedera-node/hedera-file-service-impl/src/main/java/com/hedera/node/app/service/file/impl/handlers/FileAppendHandler.java

```diff
@@ -107,6 +111,10 @@ public void handle(@NonNull final HandleContext handleContext) throws HandleExce
         }
         final var file = optionalFile.get();
 
+        // First validate this file is mutable; and the pending mutations are allowed
```

Comment:
When privileged checks are done, we'll have to check that here and skip at least the mutability check for privileged "payer" accounts

Reply:
added todo here